src/api/schema/components/mod.rs

```diff
@@ -24,6 +24,7 @@ use tokio_stream::{wrappers::BroadcastStream, Stream, StreamExt};
 #[derive(Debug, Clone, Interface)]
 #[graphql(
     field(name = "component_id", type = "String"),
+    field(name = "pipeline_id", type = "Option<&str>"),
```

Comment:
Not a big deal, but I'm curious if this needs to be an `Option<&str>` rather than `Option<String>`?

Reply:
When I change it, the compiler gives me the following error

```
error[E0277]: the trait bound `std::option::Option<std::string::String>: From<std::option::Option<&str>>` is not satisfied
   --> src/api/schema/components/mod.rs:24:24
    |
24  | #[derive(Debug, Clone, Interface)]
    |                        ^^^^^^^^^ the trait `From<std::option::Option<&str>>` is not implemented for `std::option::Option<std::string::String>`
```

So `Option<&str>` is acceptable 😃

Reply:
Aha 👍